cstrml/swork/Cargo.toml

```diff
@@ -28,14 +28,14 @@ frame-benchmarking = { version = '2.0.0', default-features = false, optional = t
 
 # crust runtime modules
 market = { version = "0.10.0", package = 'cstrml-market', path = '../market', default-features = false }
```

Comment:
will remove this line later since swork doesn't directly depend on market anymore

Reply:
sure